string event
string date
bool isWeekly
---
string resultCode
